### DISCUSSION TITLE
GPG output

### DISCUSSION THREAD

**Discussion by nefilim** [ckipp01/mill-ci-release#48] [Ideas]
Hi Chris - wondering what you think about swallowing the output of GPG (unless there's a problem?), it would be great if my email address was a little less crawlable 😅

**Comment by ckipp01** [comment]
Hey @nefilim. To be honest I haven't really thought about that too much. Mainly because:

1. I copied the logic from https://github.com/sbt/sbt-ci-release, which does the same thing. It has quite a few users and this hasn't really came up as an issue yet.
2. I understand it's a bit different, but the email is somewhat public already seeing that it's attached to the key. I'd recommend using an email you don't mind being out there for this.

All that to say, it's probably not something I'll look into atm.
